stored_requests/backends/http_fetcher/fetcher.go

```diff
@@ -34,21 +39,33 @@ import (
 //     "imp3": null // If imp3 is not found
 //   }
 // }
+// or
+// {
+//   "accounts": {
+//     "acc1": { ... config data for acc1 ... },
+//     "acc2": { ... config data for acc2 ... },
+//   },
+// }
 //
 //
 func NewFetcher(client *http.Client, endpoint string) *HttpFetcher {
 	// Do some work up-front to figure out if the (configurable) endpoint has a query string or not.
 	// When we build requests, we'll either want to add `?request-ids=...&imp-ids=...` _or_
-	// `&request-ids=...&imp-ids=...`, depending.
+	// `&request-ids=...&imp-ids=...`.
+
+	if url, err := url.Parse(endpoint); err != nil {
+		glog.Errorf(`Invalid endpoint "%s": %v`, endpoint, err)
```

Comment:
This should be `golg.Fatalf`. `Errorf` will just log the error and move on vs `Fatalf` will log the error and call `os.Exit(255)` and for an error like this, we wanna fail fast.